CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# Changelog
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/amplify-education/terrawrap/pullRequest?prid=5656512)